src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownEventListener.java

```diff
@@ -406,6 +407,9 @@ public void onTownStatusScreen(TownStatusScreenEvent event) {
 
 							// > Points: +90 / -220
 							out.add(Translation.of("status_town_siege_battle_points", siege.getFormattedAttackerBattlePoints(), siege.getFormattedDefenderBattlePoints()));
+							
+							// > Current Battle Session Reward: 420 points

```

Comment:
:construction: 
- If the value gets displayed at all, is the current text the right one to show on the display ?
- e.g. it is quite lengthy and repeats the word "battle" in the section
- How about something shorter, e.g. "Pending Balance Adjustment" ?